daemon/health.go

```diff
@@ -93,6 +93,10 @@ func (d *Daemon) initHealth() {
 				return err
 			},
 			RunInterval: 30 * time.Second,
+			// Add backoff to launching the health endpoint in the case that
+			// regenerations fail so we do not keep on retrying compilations,
+			// which are costly.
+			ErrorRetryBaseDuration: 1 * time.Second,
```

Comment:
Looking at the docs of `ErrorRetryBaseDuration`, it should already be 1 second by default?
https://github.com/cilium/cilium/blob/cda91171ccec333485bdf533fcc57b02e163b78e/pkg/controller/controller.go#L77-L81

Reply:
You are right! I'll drop the commit.